api/v1alpha1/toolchainconfig_types.go

```diff
@@ -79,6 +79,13 @@ type HostConfig struct {
 	// Keeps parameters necessary for configuring Space provisioning functionality
 	// +optional
 	SpaceConfig SpaceConfig `json:"spaceConfig,omitempty"`
+
+	// Contains the PublicViewer configuration.
+	// IMPORTANT: To provide a consistent User-Experience, each user
+	// the space has been directly shared with should have at least
+	// the same permissions the kubesaw-authenticated user has.
+	//+optional
+	PublicViewerConfig *PublicViewerConfiguration `json:"publicViewerConfig,omitempty"`
```

Comment:
is this ok or do we prefer to have `publicViewer`?
```suggestion
	PublicViewerConfig *PublicViewerConfiguration `json:"publicViewer,omitempty"`
```